source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -121,6 +121,7 @@ public ActionResult PeriodEvaluation(string period)
             var isEvaluated = evaluation.Any(p => p.UserName == loggedUser);
             ViewBag.Period = period;
             ViewBag.hasAutoCalification = isEvaluated;
+            ViewBag.isEvaluationRedyForDevolution = evaluation.Any(e => e.UserName == loggedUser && e.OpenToDevolution);
```

Comment:
Also, what do you think about to change the name to `isEvaluationOpenForDevolution`? because it is not only ready, it is also opened.